DataScience/Analytics/prototype/automatic_testing.py

```diff
@@ -65,14 +70,22 @@ def run_all_datasets():
 
     :return: None
     """
+    settings = dict(ABpath=ABpath, outpath=outpath, inputPath=inputPath)
+
     print('Running Edge Case addresses test...')
-    ec.run_edge_case_linker()
+    ec.run_edge_case_linker(**settings)
+
+    if local:
+        print('Cannot run Patient Records test locally...')
+    else:
+        print('Running Patient Records addresses test...')
+        pr.run_patient_record_address_linker(**settings)
 
     print('Running Welsh addresses test...')
-    wa.run_welsh_address_linker()
+    wa.run_welsh_address_linker(**settings)
 
     print('Running Landry Registry addresses test...')
-    lr.run_land_registry_linker()
+    lr.run_land_registry_linker(**settings)
```

Comment:
It would be better to create a class called Settings and pass an instance of it through instead of using `**` stuff, but that's debatable